alex_calc.rb

```diff
@@ -0,0 +1,79 @@
+### Calculator program ###
+
+operators_list = %w(add + subtract - multiply * divide /)
+
+# methods
+def validate_operator(operator, operators_list)
+  while !(operators_list.include?(operator))
+    print "Please enter an operator word or symbol (i.e. add, +, *): "
+    operator = gets.chomp
+  end
+  return operator
+end
+
+def validate_num(num)
+  while num =~ /[[:alpha:]]/
```

Comment:
This is only checking to see if the value is an alphabetic letter, so punctuation doesn't count.